src/main/java/task/TaskList.java

```diff
@@ -0,0 +1,186 @@
+package task;
+
+import java.util.List;
+
+import exception.DukeException;
+import storage.Storage;
+import task.tasks.Deadline;
+import task.tasks.Event;
+import task.tasks.Task;
+import task.tasks.Todo;
+
+/**
+ * Contains list of tasks and provide operations to manipulate this list of tasks.
+ */
+public class TaskList {
+    /**
+     * Task list.
+     */
+    protected List<Task> tasks;
+
+    /**
+     * Creates TaskList object containing a list of tasks.
+     *
+     * @param tasks Task list.
+     */
+    public TaskList(List<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    /**
+     * Retrieves task list from TaskList object.
+     *
+     * @return Task list.
+     */
+    public List<Task> getTasks() {
+        return this.tasks;
+    }
+
+    /**
+     * Adds a task to task list.
+     *
+     * @param userCommand User input.
+     * @throws ArrayIndexOutOfBoundsException If /by or /at user input not written properly.
+     *                                        E.g deadline return book /bylmklmlmlkmlkmlmlmlmlmkl Sunday.
+     */
+    public String addTask(String userCommand) {
+        if (userCommand.contains("todo")) { // To Do
```

Comment:
Perhaps it would be clearer if the comment contains a phrase rather than a word(e.g // Handle Todo)?